calico/maintenance/ebpf/enabling-ebpf.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Enable the eBPF dataplane
+description: Step-by-step instructions for enabling the eBPF dataplane.
+canonical_url: '/maintenance/ebpf/enabling-ebpf'
+---
+
+### Big picture
+
+Enable the eBPF dataplane on an existing cluster.
+
+### Value 
+
+{% include content/ebpf-value.md %}
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **calico/node**
+- **eBPF dataplane**
+
+### Concepts
+
+#### eBPF
+
+eBPF (or "extended Berkeley Packet Filter"), is a technology that allows safe mini programs to be attached to various low-level hooks in the Linux kernel. eBPF has a wide variety of uses, including networking, security, and tracing. You’ll see a lot of non-networking projects leveraging eBPF, but for {{site.prodname}} our focus is on networking, and in particular, pushing the networking capabilities of the latest Linux kernels to the limit.
+
+### Before you begin
+
+#### Supported
+
+- x86-64
+
+- Distributions:
+
+  - Generic or kubeadm
+  - kOps
+  - OpenShift
+  - EKS
+  - AKS with limitations:
+    - [AKS with Azure CNI and Calico network policy](../../getting-started/kubernetes/managed-public-cloud/aks#install-aks-with-{{site.prodnamedash}}-for-network-policy) works, but it is not possible to disable kube-proxy resulting in wasted resources and suboptimal performance.
+    - [AKS with {{site.prodname}} networking](../../getting-started/kubernetes/managed-public-cloud/aks#install-aks-with-{{site.prodnamedash}}-networking) is in testing with the eBPF dataplane. This should be a better solution overall but, at time of writing, the testing was not complete.
+  - RKE
+
+- Linux distribution/kernel:
+
+  - Ubuntu 20.04.
+  - Red Hat v8.2 with Linux kernel v4.18.0-193 or above (Red Hat have backported the required features to that build).
+  - Another [supported distribution]({{site.baseurl}}/getting-started/kubernetes/requirements) with Linux kernel v5.3 or above.
```

Comment:
We should probably state that we recommend 5.8+ with BTF enabled because in that case the eBPF dataplane is even more efficient.